Cliqz/MyOffrz/OffrzDataSource.swift

```diff
@@ -63,7 +62,11 @@ class OffrzDataSource {
         LocalDataStore.set(value: "closed", forKey: OffrzDataSource.OffrzOnboardingKey)
     }
 
-	private func updateMyOffrzList() {
+	func updateMyOffrzList() {
+        guard self.lastFetchDate == nil || Date().timeIntervalSince(self.lastFetchDate!) > self.expirationDuration  else {
```

Comment:
Offrz are unlike news could be empty if there were no offrz in that day for that region, in that case it will always call the endpoint and return nothing every time